src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -860,6 +860,7 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 				isSRSFlow ? self?.showSymptomsScreen() : showNextScreen()
 			},
 			onSkip: { [weak self] in
+				self?.model.exposureSubmissionService.checkins = []
```

Comment:
The code is fine. Just for a better pattern:
Please call a `viewModel` method, that calls the `exposureSubmissionService,` that reset the `checkins` array.

Reply:
Done